plugins/sqlite_dataset.cc

```diff
@@ -387,7 +388,9 @@ struct SqliteSparseDataset::Itl
                 explanation.append(record);
             }
 
-            INFO_MSG(logger) << queryStr << "\n" << explanation;
+            cerr << "rowHash = " << int64_t(RowHash(rowName)) << endl;
+            cerr << "rowName = " << rowName << endl;
+            cerr << queryStr << "\n" << explanation << endl;
```

Comment:
Seems to be the opposite of where we want to go.

Reply:
It's a pre-crash log.  We don't want to log to a logging subsystem just before we crash the system.

Reply:
Adding that as a comment in the code would be helpful.

Reply:
It does call abort() immediately after... it's pretty clear from the context.